### ISSUE TITLE
[Feature] Optional reordering when done

### ISSUE THREAD

**Issue by reggermont** [lolo-io/OneList#10]
Hi, first of all thanks a lot for your app and releasing it as Open Source, never found an app like this one which fit my needs

The only issue I have with it is that it reorders a todo element when I validate it as done

This feature is interesting for reusable lists (like a weekly todo list that I want to keep in a specific order)

It could be an option when creating the todo list, but it would impact the file save, as the sharing

**Comment by lolo-io** [comment]
Hello,
I was thinking of adding an option in the settings to chose whether the done items are reordering to the end or not, and if new items are added at the beggining or the end of the list.
The same settings would apply to all the lists.

**Comment by reggermont** [comment]
Yeah that can be a solution too, but that prevents the user to choose per list :smile:

**Comment by CurbedLarry** [comment]
Sorting options badly needed.  I have ~100 items on my shopping list, some I buy every week, some once per year.
